### PR TITLE
280 update tests

### LINKED ISSUE #280: Update tests
The tests were not supported for a long time, so some of them stopped passing. This should be fixed

### PR DESCRIPTION
**Pull request by Boneyan**
All tests (uncluding uncommented) now pass
Closes #280

### CODE REVIEW

**Comment by ikopylov** [comment]
Add a line to CHANGELOG.md describing the changes you made. This is required step for every PR.

Also, please, write all comments and text here in English

**Comment by ikopylov** [comment]
Also, please, add link to the issue in format: 'Closes https://github.com/qoollo/bob/issues/280'. This way the issue will be automatically closed when the request is merged

File: bob/src/cluster/tests.rs [hunk 1]
```diff
@@ -2,64 +2,65 @@ use std::sync::atomic::{AtomicU64, Ordering};
...
+    let cl = node;
 
-    // client
-    //     .expect_ping()
-    //     .returning(move || test_utils::ping_ok(cl.name().to_owned()));
-    todo!("mock");
+    client
+         .expect_ping()
+         .returning(move || test_utils::ping_ok(cl.name().to_owned()));
+    //todo!("mock");
```
Comment (pyakushin) [inline bob/src/cluster/tests.rs:26]: You can delete this now

Comment (pyakushin) [reply bob/src/cluster/tests.rs:26]: And other commented lines

**Review by pyakushin — APPROVED** [review]

**Pull request merged**